modules/glean/runtime/glean-plugin.client.ts

```diff
@@ -13,9 +13,11 @@ export default defineNuxtPlugin((nuxtApp) => {
     const GLEAN_APP_ID = 'moso-mastodon-web'
     const env = useAppConfig().env
     const devMode = env === ('dev' || 'canary' || 'preview')
+    const prodMode = env === 'release'
```

Comment:
sorry I may be out of the loop on this; does `useAppConfig().env` distinguish between stage and prod? What is the env value for stage?

Reply:
Good question @aaga! I assumed that it was able to distinguish between stage & production, but I haven't verified that yet. I reached out to Kirill to confirm whether his team is seeing events coming from the `dev` channel, and if so, he should be able to tell us which if that `env` is getting set for stage as well.

In the meantime, I'm also looking into `useBuildInfo()` and whether that might be what we want.

Reply:
Did some digging here and it looks like `useBuildInfo` just uses `useAppConfig` under the hood, so I'm not feeling super confident that either of these work the way I think they do. In order to help test this, I've removed the environment check when initializing glean, but passing through the `env` that gets set from `useBuildInfo` as the `channel` - this will allow us to learn from the data team what that `env` variable is getting set as in staging & production. After that we can add back in the environment check if we want to.